pepper-apis/src/main/java/org/broadinstitute/ddp/route/UserActivityInstanceListRoute.java

```diff
@@ -60,6 +60,7 @@ public UserActivityInstanceListRoute(ActivityInstanceDao activityInstanceDao) {
             // IMPORTANT: do numbering before filtering so each instance is assigned their correct number.
             performActivityInstanceNumbering(summaries);
             summaries = filterActivityInstancesFromDisplay(summaries);
+            activityInstanceDao.countActivitySummaryQuestionsAndAnswers(handle, userGuid, studyGuid, summaries);
```

Comment:
Some of the instances will get filtered out if they are hidden or otherwise not shown. So the key insight here is to doing the counting after the filtering, so to save us some work.